### PR TITLE
[Tester] fix fork generator

### PR DESCRIPTION
**Pull request by AskAlexSharov**
- fix coinbase 
- use same genBlock function
- switch inside genBlock - now works on blockNumber instead of nonce
- fill file in own function
- add verbosity cli flag

### CODE REVIEW

File: ethdb/remote/remotedbserver/server.go [hunk 1]
```diff
@@ -526,7 +526,7 @@ func Listen(ctx context.Context, ln net.Listener, db ethdb.HasKV) {
 	defer close(ch)
 
 	go func() {
-		ticker := time.NewTicker(3 * time.Second)
+		ticker := time.NewTicker(30 * time.Second)
```
Comment (mandrigin) [inline ethdb/remote/remotedbserver/server.go]: why this change?

Comment (AskAlexSharov) [reply ethdb/remote/remotedbserver/server.go]: No, reason. I will revert now.
